src/Metadata/MetadataBase.php

```diff
@@ -220,23 +220,13 @@ public function getRole(): string
     }
 
     /**
-     * @return boolean
-     *    Whether the metadata is trusted.
-     */
-    public function isTrusted(): bool
-    {
-        return $this->isTrusted;
-    }
-
-    /**
-     * @param boolean $isTrusted
-     *   Whether the metadata should be trusted.
+     * Sets the metadata as trusted.
      *
      * @return void
      */
-    public function setIsTrusted(bool $isTrusted): void
+    public function setTrusted(): void
```

Comment:
I am only up for name change since this isn't really a getter but rather more like a 1 way switch. Maybe `markAsTrusted()` ?

Reply:
Maybe just `trust()`?

Reply:
that works. fixed